static/app/views/organizationStats/index.tsx

```diff
@@ -331,6 +331,7 @@ const DropdownDataCategory = styled(DropdownControl)`
   grid-column: auto / span 1;
   justify-self: stretch;
   align-self: stretch;
+  z-index: 1;
```

Comment:
Oh, I was hoping you'd fix this everywhere, because it's actually broken in the admin stuff im doing too
![image](https://user-images.githubusercontent.com/1421724/130691539-a5595c11-7664-47ae-b2ea-a0be1f02dcc1.png)

Reply:
Would it be better to centralize this here? https://github.com/getsentry/sentry/blob/5e310d83115e06c4558e11fa9232eef32f08500e/static/app/utils/theme.tsx#L479-L539

Reply:
Yeah that's what I was thinking too